backbone/src/main/java/org/researchstack/backbone/ui/step/layout/SurveyStepLayout.java

```diff
@@ -350,4 +361,21 @@ private void checkIfAllStepsAreOptional() {
     public StepBody getStepBody() {
         return stepBody;
     }
+
+    public void revertAllChildren(@NotNull StepResult originalResult) {
```

Comment:
The same goes for this method... I'd call it: setFormStepChildrenResultsTo(...) (or similar, you think of what this method really does. It's not "Reverting".

Also does this need to be `public`? 

Finally, any way to Unit Test this behavior? (that the method does what it says it does?)

Reply:
No it's not need. I will change this too.

For the test to be honest i didn't even think about it ^^' I will try to test this.